cmd/cloud-api-adaptor/cloudmgr/vsphere.go

```diff
@@ -23,16 +23,16 @@ type vsphereMgr struct{}
 func (_ *vsphereMgr) ParseCmd(flags *flag.FlagSet) {
 
 	flags.StringVar(&vspherecfg.VcenterURL, "vcenter-url", "", "URL of vCenter instance to connect to")
-	flags.StringVar(&vspherecfg.UserName, "user-name", "", "Username, defaults to `GOVC_USERNAME`")
-	flags.StringVar(&vspherecfg.Password, "password", "", "Password, defaults to `GOVC_PASSWORD`")
+	flags.StringVar(&vspherecfg.UserName, "user-name", "", "vCenter Username")
+	flags.StringVar(&vspherecfg.Password, "password", "", "vCenter Password")
 	flags.StringVar(&vspherecfg.Thumbprint, "thumbprint", "", "SHA1 thumbprint of the vcenter certificate. Enable verification of certificate chain and host name.")
 	flags.StringVar(&vspherecfg.Template, "template", "podvm-template", "vCenter template to deploy")
 	flags.StringVar(&vspherecfg.Datacenter, "data-center", "", "vCenter desination datacenter name")
-	flags.StringVar(&vspherecfg.Vcluster, "vcluster-name", "", "vCenter desination cluster name for DRS placement")
 	flags.StringVar(&vspherecfg.Datastore, "data-store", "", "vCenter datastore")
-	flags.StringVar(&vspherecfg.Resourcepool, "resource-pool", "", "vCenter desination resource pool")
 	flags.StringVar(&vspherecfg.Deployfolder, "deploy-folder", "", "vCenter vm desintation folder relative to the vm inventory path (your-data-center/vm). \nExample '-deploy-folder peerods' will create or use the existing folder peerpods as the \ndeploy-folder in /datacenter/vm/peerpods")
-
+	flags.StringVar(&vspherecfg.Cluster, "cluster", "", "vCenter desination cluster name ")
+	flags.StringVar(&vspherecfg.DRS, "drs", "false", "Use DRS for clone placement in desination Vcenter cluster")
```

Comment:
You can also use a BoolVar. If the option `-drs` is missing it's `false`, if the option is present then it's `true`.